### ISSUE TITLE
Can't find TDS_Introduction_sine_waves_complex64.bin file

### ISSUE THREAD

**Issue by DDubbleDD** [potto216/tds-tutorials#6]
Tried to run the TDS_Introduction_sine_waves Gnu Radio file and the TDS_Introduction_sine_waves_complex64.bin file is missing.  Didn't find it in the .Zip file or GitHub.

**Comment by potto216** [comment]
Thanks, I forgot to include the documentation the grc file about how to generate the data file from notebooks/TDS_Introduction-sine_waves.ipynb. I'll fix that and correct a path issue.

**Comment by potto216** [comment]
I pushed changes explain how to generate the data file and checks to verify the data file exists and print an error method with solution suggestions if it doesn't. Let me know how this works for you

**Comment by DDubbleDD** [comment]
Hi Paul,
I got this to work.  I stopped using the Anaconda installation and have switched over to Visual Studio Code and stock Python.  The implementation of Jupyter Lab is not quite as complete with this setup, so I had to switch the first line of the Jupyter notebooks from %matplotlib notebook to %matplotlib inline.  Then that seemed to work after making sure the path for the file was valid.

Thanks for the help and the great content.
